web-frontend/src/main.tsx

```diff
@@ -13,15 +13,20 @@ import SelectAccount from './pages/SelectAccount/SelectAccount.tsx';
 import Error from './pages/Error.tsx';
 import AddAccount from './pages/AddAccount/AddAccount.tsx';
 import { QueryClient, QueryClientProvider } from '@tanstack/react-query';
+import mockServer from './mirage/server.ts';
+
+if (import.meta.env.VITE_ENV === 'dev') {
+  mockServer();
+}
 
 const queryClient = new QueryClient();
 
 const router = createBrowserRouter(
   createRoutesFromElements(
-    <Route path={import.meta.env.VITE_BASE_PATH}>
-      <Route path="index" element={<Welcome />} />
+    <Route path={import.meta.env.VITE_BASE_APP}>
+      <Route path="welcome" element={<Welcome />} />
```

Comment:
keeping "/" or index for the welcome page is nice ? isnt it?

Reply:
it's not possible, goswagger will not fallback into the /web-app/{ressource} route

Reply:
we can keep as it was if you think index is better than welcome